common-libs/common/test/factory/CommonNetworkMember.ts

```diff
@@ -0,0 +1,26 @@
+import { EventComponent } from '@affinidi/affinity-metrics-lib'
+import { CommonNetworkMember as GenericCommonNetworkMember, __dangerous } from '@affinidi/wallet-core-sdk'
```

Comment:
@inga-affinidi so we decided still keep core-sdk as dev dependency at the common lib? Ok, I am fine (its could be another task, scope to get rid from it here)

Reply:
Yes, it should be another task, I've just reorganized code a bit in `common` tests, nothing of substance has changed.